src/main/java/ru/sbtqa/tag/pagefactory/support/MobileConsole.java

```diff
@@ -0,0 +1,42 @@
+package ru.sbtqa.tag.pagefactory.support;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.sbtqa.tag.pagefactory.drivers.TagMobileDriver;
+
+public class MobileConsole {
+    
+    private static final Logger LOG = LoggerFactory.getLogger(MobileConsole.class);
+    
+    public static boolean execute(String command) {
+        return execute(TagMobileDriver.getDeviceUDID(), command);
+    }
+
+    public static boolean execute(String deviceUDID, String command) {
+        ProcessBuilder processBuilder = new ProcessBuilder(new String[]{"adb", "-s", deviceUDID, "shell", command});
+        LOG.info("Command '{}' is processing...", command);
+        Process process;
+        try {
+            process = processBuilder.start();
+
+            BufferedReader reader
+                    = new BufferedReader(new InputStreamReader(process.getInputStream()));
+            StringBuilder builder = new StringBuilder();
+            String line;
+            while ((line = reader.readLine()) != null) {
+                builder.append(line);
+                builder.append(System.getProperty("line.separator"));
+            }
+            LOG.debug(builder.toString());
+            
+            return (process.waitFor() == 0);
+        } catch (IOException | InterruptedException ex) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)